style/tailwind.css

```diff
@@ -265,3 +276,45 @@ pre[class*="language-"] {
 .highlight pre {
   margin: 0;
 }
+.entry-content p {
+  font-size: 18px;
+}
+.entry-content img {
+  margin: 30px auto;
+  border-radius: 6px;
+}
+@media only screen and (max-width: 769px) {
```

Comment:
Icons are vertical and on the left side of the window, and are horizontal and on the bottom on mobile view.

Reply:
didn't see them on mobile view.

Reply:
![Screen Shot 2022-06-22 at 12 12 16 pm](https://user-images.githubusercontent.com/72968321/174928481-7eb4b09d-2d49-4f0a-9ac8-118a9727bf6f.png)
It shows up here, next I will be working on some mobile designs, mobile font-size, nav is not showing in mobile view I will fix that and look this issue as well.

Reply:
Ok